src/App.js

```diff
@@ -1,84 +1,99 @@
 import React, { Component } from 'react';
 import './App.css';
-
-import Home from './component/Home';
-import Completed from './component/Completed';
-import Remaining from './component/Remaining';
+import Container from './component/Container';
+import AddTodoInput from './component/AddTodoInput';
+import Header from './component/Header';
+import Tabs from './component/Tabs';
 
 class App extends Component {
+
   constructor() {
     super();
     this.state = {
-      todoList: [
-        {
-          id: 123,
-          content: 'create react app',
-          completed: false
-        },
-        {
-          id: 124,
-          content: 'create java app',
-          completed: true
-        }
-      ],
+      todoList: [],
       currentView: 'home'
```

Comment:
It seems you are representing the active window as `Tab` so I think it would make more sense if we change the variable name to `currentTab` or `activeTab`.